### DISCUSSION TITLE
Quick Deployment: Virtual Machines

### DISCUSSION THREAD

**Discussion by twoex** [YetiForceCompany/YetiForceCRM#17279] [Wishlist]
Would be amazing if there was a downloadable version of a preconfigured Yetiforce instance that works with VMWare, Virtualbox, self contained Windows install etc for quick deployment, whether for testing, tinkering, development or even a live intranet instance ....

would be great to be able to quickly deploy a copy of YF... or even creating installation scripts for common distros, etc...

**Comment by ghost** [comment]
They have an install guide for debian 10 on their website, though it works for debian 12 aswell. [https://doc.yetiforce.com/developer-guides/environments/debian-10/](https://doc.yetiforce.com/developer-guides/environments/debian-10/)

Their [dockerfile](https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/Dockerfile) can also be used to easily create a script, or just install the docker itself

I also have a personal install script which installs yetiforce with a few tweaks, though its based on the attached urls.
